docs/notebooks/Training_and_inference_using_Google_Drive.ipynb

```diff
@@ -1,543 +1,300 @@
-        "\n",
-        "Let's inspect the predictions file:"
-      ]
-    },
-    {
-      "cell_type": "code",
-      "metadata": {
-        "id": "nPfmNMSt-vS7"
-      },
-      "source": [
-        "!sleap-inspect colab_demo.mp4.predictions.slp"
-      ],
-      "execution_count": null,
-      "outputs": []
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "LoJ2kNBK-w6k"
-      },
-      "source": [
-        "You can copy this file from your Google Drive to a local drive and open it in the SLEAP GUI app (or open it directly if you have your Google Drive mounted on your local machine). If the video is in the same directory as the predictions file, SLEAP will automatically find it; otherwise, you'll be prompted to locate the video (since the path to the video on your local machine will be different than the path to the video on Colab)."
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "Mf-MsD7fv5Wy"
-      },
-      "source": [
-        "## Inference parameters\n",
-        "\n",
-        "One important option when running inference is whether (and how) you want to track instance identities. If you omit `--tracking.tracker flow` then the identities will not be tracked. Tracking methods/options are explained [here](https://sleap.ai/guides/proofreading.html#tracking-methods).\n",
-        "\n",
-        "You can see all of the command-line arguments by calling `sleap-track` with the `--help` argument, like so:"
-      ]
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "iq7jrgUksLtR"
+   },
+   "source": [
+    "# Impoprt training data into Colab with Google Drive\n",
+    "We'll first prepare and export the training data from SLEAP, then upload it to Google Drive, and then mount Google Drive  into this Colab notebook."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "GwpEwrxYdLMR"
+   },
+   "source": [
+    "## Create and export the training job package\n",
+    "A self-contained **training job package** contains a .slp file with labeled data as well as the labeled images which will be used for training. One advantage to exporting a training job package is that it doesn't depend on paths to other files (i.e., videos) to be messed up when you copy your project to another volume.\n",
+    "\n",
+    "See [this guide](https://sleap.ai/guides/training-job-package.html) for exporting a training job package from SLEAP."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "ApaDWxW4dLMS"
+   },
+   "source": [
+    "## Upload training job package to Google Drive\n",
+    "To be consistent with the examples in this notebook, name the SLEAP project `colab` and create a directory called `sleap` in the root of your Google Drive. Then upload the exported training job package `colab.slp.training_job.zip` into `sleap` directory.\n",
```

Comment:
See comment in the next notebook, but this is a little weird here... when you export the package in the zip file, it includes the `.pkg.slp` file, as well as the training jobs, which will just be ignored here?

Might it make more sense to be explicit about this and/or just tell people to use the .pkg.slp file (also exportable without needing to go into the training editor)?